lib/factory.js

```diff
@@ -56,7 +58,6 @@ export default class Factory {
 
     create(cb) {
         this.promise
-            .then(store => Handlers.resolveImport(store, this.basedir))
```

Comment:
Wondering if there is ever a time that this would still be handy?
`npm start --services='import:overrides.json'`